packit_service/worker/handlers/koji.py

```diff
@@ -293,7 +295,7 @@ def notify_about_failure_if_configured(
 
 
 @reacts_to_as_fedora_ci(event=koji.result.Task)
-class KojiTaskReportDownstreamHandler(AbstractKojiTaskReportHandler):
+class KojiTaskReportDownstreamHandler(AbstractKojiTaskReportHandler, KojiBuildHandler):
```

Comment:
I really don't think this is a good idea, just use `GetKojiBuildJobHelperMixin` if you really need `koji_build_helper`.